packages/contracts/scripts/test_userOp_testnet.ts

```diff
@@ -17,24 +18,26 @@ import { toEcdsaKernelSmartAccount } from "permissionless/accounts"
 import { type Erc7579Actions, erc7579Actions } from "permissionless/actions/erc7579"
 import { createPimlicoClient } from "permissionless/clients/pimlico"
 import { deployment as happyTestnetDeployment } from "../deployments/happy-sepolia/aa/abis"
+import * as fs from 'node:fs';
 
 // Environment detection
 const privateKey = process.env.PRIVATE_KEY_TEST as Hex
-const bundlerRpc = "https://bundler-staging.happy.tech"
+const bundlerRpc = "http:localhost:49531"//"https://bundler-staging.happy.tech"
```

Comment:
i am running a testnet bundler locally, from another shell

Reply:
if the staging hosted bundler is updated with the above `config.happySepolia.json` can use that